content/maxwell3_fdem/sphere_freespace/dipole_response.rst

```diff
@@ -0,0 +1,91 @@
+
+Consider the problem geometry illustrated in :numref:`FreeSpaceSphere`, wherein a sphere of radius :math:`R`, conductivity :math:`\sigma` and magnetic permeability :math:`\mu` is located in the vicinity of an inductive source transmitter (Tx).
+The transmitter generates a harmonic primary field :math:`{\bf H_0} (i\omega)` which induces an excitation within the sphere.
+Because we are in free-space, :math:`{\bf H_0} (i\omega)` may be calculated using the Biot-Savart law (link).
+The excitation induced within the sphere produces a secondary field :math:`{\bf H_s} (i\omega)` which is then measured by a receiver coil (Rx).
+
+
+.. figure:: ./images/figDipoleField.png
+        :align: right
+        :figwidth: 40%
+        :name: DipoleField
+        
+        Response from a conductive and magnetically permeable sphere, under the influence of a uniform, harmonic inducing field :math:`{\bf H_0} (i\omega)`. The induced dipole moment is represented by :math:`{\bf m} (i\omega)` and the resulting dipole response is represented by :math:`{\bf H} (i\omega)`.
+
+To approximate the sphere's response, Wait (1951) considered the dipole response from a conductive and magnetically permeable sphere under the influence of a uniform, harmonic field (:numref:`DipoleField`).
+For a magnetic dipole moment :math:`{\bf m} (i\omega)`, the dipole field :math:`{\bf H} (i\omega)` is given by:
+
+.. math::
+	{\bf H} (i \omega) =\frac{1}{4\pi} \Bigg [ \frac{3 {\bf r} \; \big [ {\bf m}(i \omega) \cdot {\bf r} \; \big ]}{r^5} - \frac{{\bf m} (i \omega) }{r^3} \Bigg ]
+	:label: eqDipoleVacuum
+
+where :math:`{\bf r}` is the vector distance between :math:`P` and :math:`Q`.
+For practical problems, the inducing field may be considered approximately uniform about the sphere if 1) the radius of the sphere is much smaller than the wavelength of the inducing field , and 2) the distance between the transmitter and the sphere is sufficiently larger than the sphere's radius (typically :math:`> 10R` ).
+If these conditions are satisfied, the magnetic dipole moment characterizing the sphere's excitation is given by:
+
+.. math::
+	{\bf m} (i \omega) = \frac{4\pi}{3}R^3 \chi (i \omega) \, {\bf H_0} (i \omega)
+	:label: eqDipoleMoment
+
+where :math:`\chi (i\omega)` is defined as the sphere's excitation factor.
+The dipole moment is therefore a product of the sphere's volume, its excitation factor, and the inducing field.
+The excitation factor is used to characterize the nature of the sphere's frequency-domain response.
+An explicit expression for the excitation factor is given by (Wait, 1951):
+
+.. math::
+	\chi (i \omega) = \frac{3}{2} \Bigg [ \! \frac{2\mu \big [ tanh(\alpha) - \alpha  \big ] + \mu_0 \big [\alpha^2 \, tanh(\alpha) - \alpha + tanh(\alpha) \big ] }{\mu  \big [ tanh(\alpha) - \alpha \big ] - \mu_0 [ \alpha^2 \, tanh(\alpha) - \alpha + tanh(\alpha) \big ] } \! \Bigg ]
+	:label: eqExcitationFactor
+	
+where
+
+.. figure:: ./images/figChiOmega.png
```

Comment:
a couple annotations pointing out important features would be helpful. Check out https://www.monosnap.com/welcome